htdocs/takepos/index.php

```diff
@@ -57,8 +57,8 @@
 $setterminal = GETPOST('setterminal', 'int');
 $setcurrency = GETPOST('setcurrency', 'aZ09');
 
-if ($_SESSION["takeposterminal"] == "") {
-	if ($conf->global->TAKEPOS_NUM_TERMINALS == "1") {
+if (isset($_SESSION["takeposterminal"]) && $_SESSION["takeposterminal"] == "") {
```

Comment:
if $_SESSION["takeposterminal"] is not defined the test 
`if ($_SESSION["takeposterminal"] == "") {`
will be true. To keep same behavior, you replace with
`if (!isset($_SESSION["takeposterminal"]) || $_SESSION["takeposterminal"] == "") {`